node/tessel-export.js

```diff
@@ -249,30 +249,40 @@ Tessel.Port = function(name, socketPath, board) {
 
   this.pwm = [];
 
-  this.I2C = function I2C(address, mode) {
-    return new Tessel.I2C({
+  this.I2C = function I2CInit(address, mode) {
+    if (port._i2c) {
+      port._i2c.disable();
+    }
+
+    port._i2c =  new Tessel.I2C({
```

Comment:
There can be 127 of these! I recommend reverting all of the changes made to I2C in this patch.